content/en/admin/scaling.md

```diff
@@ -412,10 +412,16 @@ REPLICA_DB_PORT
 REPLICA_DB_NAME
 REPLICA_DB_USER
 REPLICA_DB_PASS
+REPLICA_PREPARED_STATEMENTS
+REPLICA_DB_TASKS
 ```
 
 Alternatively, you can also use `REPLICA_DATABASE_URL` if you want to configure them all using the same variable.
 
+`REPLICA_DB_TASKS` will connect to an replica database without any database mangement tasks such as schema management, migrations, seeds, etc. By default it is set to true.
```

Comment:
Isn't this the opposite? Reading this comment, I'm under the impression that `REPLICA_DB_TASKS=true` means the database management tasks don't apply to the replica.

Reply:
I've added `=false` to make the statement less confusing.